src/ClassicUO.Client/Game/UI/Gumps/GridLootGump.cs

```diff
@@ -222,36 +222,44 @@ protected override void UpdateContents()
             int line = 1;
             int row = 0;
 
-            for (LinkedObject i = _corpse.Items; i != null; i = i.Next)
+            for (int displayGroup = 0; displayGroup < 2; displayGroup++)
             {
-                Item it = (Item) i;
-
-                if (it.IsLootable)
+                for (LinkedObject i = _corpse.Items; i != null; i = i.Next)
                 {
-                    GridLootItem gridItem = new GridLootItem(it, GRID_ITEM_SIZE);
+                    Item it = (Item)i;
 
-                    if (x >= MAX_WIDTH - 20)
+                    if (!ItemBelongsToGroup(it, displayGroup))
                     {
-                        x = 20;
-                        ++line;
+                        continue;
+                    }
 
-                        y += gridItem.Height + 20;
+                    if (it.IsLootable)
```

Comment:
probably better to invert the `if` statement and continue [readability stuff]

Reply:
Done.